sql/indicator_functions.sql

```diff
@@ -471,3 +471,35 @@ WHERE
     RETURN (result);
 END$$ 
 DELIMITER ;
+
+DROP FUNCTION IF EXISTS PECG_Indicator19;
+
+DELIMITER $$
+CREATE FUNCTION PECG_Indicator19(
+    p_startDate DATE,
+    p_endDate DATE,
+    p_startAge INT(11),
+    p_endAge INT (11),
+    p_includeEndAge TINYINT(1),
+    p_gender VARCHAR(1)) RETURNS INT(11)
+    DETERMINISTIC
+BEGIN
+    DECLARE result INT(11) DEFAULT 0;
+
+SELECT
+    COUNT(DISTINCT pat.patient_id) INTO result
+FROM
+    patient pat
+WHERE
+    patientGenderIs(pat.patient_id, p_gender) AND
+    patientAgeWhenRegisteredForHivProgramIsBetween(pat.patient_id, p_startAge, p_endAge, p_includeEndAge) AND
+    patientHasEnrolledIntoHivProgramDuringOrBeforeReportingPeriod(pat.patient_id, p_endDate) AND
```

Comment:
@IipperMangesh Could you please use `patientHasEnrolledIntoHivProgramDuringReportingPeriod` here?  Because we shouldn't include that patient that registered before the reporting period.
![image](https://user-images.githubusercontent.com/22587098/60085070-f53eb100-9738-11e9-8606-cf9b9b6e820d.png)